packages/@adobe/spectrum-css-temp/components/dropdown/skin.css

```diff
@@ -45,6 +45,9 @@ governing permissions and limitations under the License.
         color: var(--spectrum-dropdown-icon-color-disabled);
       }
     }
+    &:active .spectrum-Dropdown-label {
+      color: var(--spectrum-dropdown-text-color-disabled);
+    }
```

Comment:
hrm, out of curiosity did we figure out why this wasn't in spectrum css? I took a look and their picker css has a slightly different shape but they don't have this. Perhaps this bug exists in their css?

Reply:
I did my best to recreate this on spectrum-css and I don't see it. I though Rob identified the CSS that caused this was code we added. I looked at their code and ours is out of sync significantly. They've migrated from dropdown to picker classnames. It looks like they wrote the CSS in such a way to avoid this. https://github.com/adobe/spectrum-css/blob/main/components/picker/skin.css#L60-L72 I could try brining ours closer to this.

Reply:
Gotcha, IMO think pull can go in as is, just was curious if we had found a spectrum css bug or not.